src/Component/Movies.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { Component } from "react";
+import '../App.css'
+
+
+class Movies extends Component{
+    constructor() {
+        super()
+        this.state = {
+            movies: []
+        }
+    }
+fetchFilms = () => {
+    fetch("https://ghibliapi.herokuapp.com/films")
+        .then((res) => res.json())
+        .then((data)=>{
+            this.setState({
+                movies: data,
+            })
+        });
+};
```

Comment:
```suggestion
fetchFilms = () => {
    fetch("https://ghibliapi.herokuapp.com/films")
        .then((res) => res.json())
        .then((data)=>{
            this.setState({
                movies: data,
            })
        });
};

componentDidMount = () => {
     this.fetchFilms()
}
```
Remember, `fetchFilms` won't actually run until you call it. In this case, you want all of your data to be fetched immediately, so you should call it inside of `componentDidMount`.